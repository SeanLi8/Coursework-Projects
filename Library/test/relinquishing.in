add collection books.csv
add member Sean Li
add member Neas Il
rent 100000 111122
rent 100000 111121
rent 100001 111125
rent 100001 111126
rent 100001 111127
book history 111122
book history 111121
book history 111125
book history 111126
book history 111127
member books 100000
member books 100001
member history 100000
member history 100001
relinquish 999999 111122
relinquish 100000 999999
relinquish 100000 1111127
book history 111122
book history 111121
book history 111125
book history 111126
book history 111127
member books 100000
member books 100001
member history 100000
member history 100001
relinquish 100000 111122
book history 111122
book history 111121
book history 111125
book history 111126
book history 111127
member books 100000
member books 100001
member history 100000
member history 100001
relinquish all 999999
relinquish all 100001
book history 111122
book history 111121
book history 111125
book history 111126
book history 111127
member books 100000
member books 100001
member history 100000
member history 100001
exit



