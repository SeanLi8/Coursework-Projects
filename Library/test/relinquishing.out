user: 26 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: No rental history.

user: No rental history.

user: No rental history.

user: No rental history.

user: No rental history.

user: Harry Potter 5 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: No rental history for member.

user: No rental history for member.

user: No such member in system.

user: No such book in system.

user: No such book in system.

user: No rental history.

user: No rental history.

user: No rental history.

user: No rental history.

user: No rental history.

user: Harry Potter 5 (J.K. Rowling)
Harry Potter 4 (J.K. Rowling)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: No rental history for member.

user: No rental history for member.

user: Success.

user: 100000

user: No rental history.

user: No rental history.

user: No rental history.

user: No rental history.

user: Harry Potter 4 (J.K. Rowling)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Harry Potter 5 (J.K. Rowling)

user: No rental history for member.

user: No such member in system.

user: Success.

user: 100000

user: No rental history.

user: 100001

user: 100001

user: 100001

user: Harry Potter 4 (J.K. Rowling)

user: Member not currently renting.

user: Harry Potter 5 (J.K. Rowling)

user: Biggles in the Battle (W.E. Johns)
Aeneid (Virgil)
Halloween (Jamie Lee Curtis)

user: Ending Library process.
